app/[id]/page.tsx

```diff
@@ -1,11 +1,11 @@
-import Image from 'next/image';
-import { Marked } from 'marked';
-import { markedHighlight } from 'marked-highlight';
-import hljs from 'highlight.js';
-import styles from '../../styles/Home.module.scss';
-import { fetchIssuePageData } from '../../lib/github';
-import avatar from '../avatar.png';
-import { Time } from '../time-ago';
+import Image from "next/image";
+import { Marked } from "marked";
+import { markedHighlight } from "marked-highlight";
+import hljs from "highlight.js";
+import styles from "../../styles/Home.module.scss";
+import { fetchIssuePageData } from "../../lib/github";
+import avatar from "../avatar.png";
+import { Time } from "../time-ago";
```

Comment:
not sure why prettier changed formatting styles. Doesn't seem like there is a checked in prettier config but maybe we want that?